packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -98,6 +94,170 @@ export = {
           }),
         );
 
+        test.done();
+      },
+    },
+    'when a listener is added with timeout': {
+      'should add the listener timeout to the resource'(test: Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
+
+        // WHEN
+        const mesh = new appmesh.Mesh(stack, 'mesh', {
+          meshName: 'test-mesh',
+        });
+
+        new appmesh.VirtualNode(stack, 'test-node', {
+          mesh,
+          dnsHostName: 'test',
+          listeners: [appmesh.VirtualNodeListener.grpc({
+            port: 80,
+            timeout: {
+              idle: cdk.Duration.seconds(10),
+              perRequest: cdk.Duration.seconds(10),
+            },
+          })],
+        });
+
+        // THEN
+        expect(stack).to(
+          haveResourceLike('AWS::AppMesh::VirtualNode', {
```

Comment:
These should be on the same line, because otherwise the indentation is weird:

```suggestion
        expect(stack).to(haveResourceLike('AWS::AppMesh::VirtualNode', {
```

Same comment for all other added tests in this PR.